sklearn/tests/test_docstring_parameters.py

```diff
@@ -240,7 +240,7 @@ def test_fit_docstring_attributes(name, Estimator):
                'MultiTaskLasso', 'MultiTaskLassoCV',
                'OrthogonalMatchingPursuit',
                'PLSCanonical', 'PLSRegression', 'PLSSVD',
-               'PassiveAggressiveClassifier', 'RBFSampler'}
+               'PassiveAggressiveClassifier'}
```

Comment:
Uhm did you forgot to commit the file with the missing attribute in RBFSampler?

Reply:
Actually, in the beginning we thought it was a mistake to have RBFSampler in the list, because running the test only on RBFSampler, it passes. Then I noticed the inherited attribute 'n_features_in_', which was missing and invisible to the test function. I added it now.